docs/references/index.md

```diff
@@ -3,16 +3,31 @@ id: index
 title: Temporal Platform references
 ---
 
-[SDK metrics reference](/references/sdk-metrics)
+### Temporal SDK API references
+
+The Temporal SDKs provide a framework for Temporal Application development in the following languages:
+
+- [Java](https://www.javadoc.io/doc/io.temporal/temporal-sdk/latest/index.html)
+- [Go](https://pkg.go.dev/go.temporal.io/sdk)
+- [TypeScript](https://typescript.temporal.io)
+- [Python](https://python.temporal.io)
+
+The Temporal SDKs contain the following tools:
 
-[Commands reference](/references/commands)
+- A Temporal Client to communicate with a Temporal Cluster
+- APIs to use within your Workflows
+- APIs to create and manage Worker Entities and Worker Processes
 
-[Events reference](/references/events)
+### SDK metric reference
+
+The following reference lists available metrics for use by the SDKs and your Temporal application:
+
+[SDK metrics reference](/references/sdk-metrics)
 
-[Go SDK reference](https://pkg.go.dev/go.temporal.io/sdk)
+### Command and Events references
```

Comment:
Not sure this fits here.

Reply:
For some reason I thought I was commenting on "what is a Temporal SDK?" - which I am not.